README.md

```diff
@@ -1 +1,67 @@
-# first-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# first-project
+
+> building my first project to test understanding of github flow.
+
```

Comment:
- Please give the name of the project as "Hello Microverse" instead of "first-project".
- [OPTIONAL] You can make the description a bit more detail

Reply:
@aboongm  thanks a lot man I really appreciated your feedback and I totally agree with you and I already did the changes you mentioned above.